### PR TITLE
London Class 7- Yasemin Bayraktutan-JavaScript Core 1- Week2

### PR DESCRIPTION
**Pull request by ybayraktutan**
exercises comleted

# Your Details

- Your Name: Yasemin Bayraktutan 
- Your City: London
- Your Slack Name: Yasemin

# Homework Details

- Module:  JavaScript Core 1-
- Week: 2

### CODE REVIEW

File: exercises/G-conditionals-2/exercise-3.js [hunk 1]
```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if(mark >= 80){
+    return "A";
+  } else if (mark < 80 && mark > 60) {
+    return "B";
+  } else if (mark <= 60 && mark >50) {
```
Comment (akbasmustafa) [inline exercises/G-conditionals-2/exercise-3.js:16]: my explanation to '...no lower than 50...' that is should be cover "equal and greater than 50"

**Review by akbasmustafa — CHANGES_REQUESTED** [review]

File: exercises/D-predicates/exercise.js [hunk 2]
```diff
@@ -6,10 +6,14 @@
...
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
```
Comment (AhmadJanah) [inline exercises/D-predicates/exercise.js:14]: Hi Yasemin,

You did great work, congratulation,

I think between should include the numbers of 0, and 10 as well, what do you think?

Regards,

File: exercises/G-conditionals-2/exercise-3.js (hunk 1 above)
Comment (AhmadJanah) [inline exercises/G-conditionals-2/exercise-3.js:16]: Your work here is definitely correct, but you can make it shorter by removing some code, for example you can remove  (mark < 80 && ) from second if, as the compiler will come here only when mark < 80. your code can be:
if(mark >= 80){
    return "A";
  } else if (mark > 60) {
    return "B";
  } else if (mark >50) {

**Review by AhmadJanah — COMMENTED** [review]
Thank you Yasemin for this great work, I hope you will accept my comments leniency.
Your code is amazing, but I added some extra approvments.

Best Wishes,

File: exercises/G-conditionals-2/exercise-3.js (hunk 1 above)
Comment (ybayraktutan) [reply exercises/G-conditionals-2/exercise-3.js:16]: Thank you very much for your suggestion. I will amend my code.

Comment (ybayraktutan) [reply exercises/G-conditionals-2/exercise-3.js:16]: Thank you very much for your suggestion. I will amend my code.

**Pull request closed without merging**